plugins/videobasedtracker/BeaconBasedPoseEstimator.cpp

```diff
@@ -115,12 +117,16 @@ namespace vbtracker {
         m_permittedOutliers = permittedOutliers;
     }
 
-    bool BeaconBasedPoseEstimator::SetBeacons(const Point3Vector &beacons) {
-        return SetBeacons(beacons, DEFAULT_MEASUREMENT_VARIANCE);
+    bool BeaconBasedPoseEstimator::SetBeacons(
+        const Point3Vector &beacons,
```

Comment:
`const` should follow typename (though I think clang-format will sort that out).